docs/03-computation/03-how-it-works.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: How It Works
+sidebar_position: 2
+tags: ['explainer']
+---
+
+Based on the configuration you provided, the Privacy Computation Engine infers a set of **Processing Records** that contain:
+
+- Identification of the data field in your database. Table/Object and field/property. E.g. `USER`, `e_mail_address`
+- A [Privacy Scope](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#privacy-scope) e.g. (`CONTACT.EMAIL` x {`COLLECTION`, `STORING`, `USING`} x {`SERVICES`,`COMPLIANCE`})
+- One or more [Legal Bases](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#legal-base) e.g. a legal base of type `CONTRACT`
+
+It is possible to end-up having several records for one Data Category, e.g.
+
+<!-- prettier-ignore -->
+| Data in database | Privacy Scope | Legal bases |
+| :----: | :-----: | :--------|
+| table:`USER` field:`name` | `NAME.FIRST-NAME` x {`COLLECTION`, `STORING`, `USING`} x {`SERVICES`,`COMPLIANCE`} | `CONTRACT` |
+| table:`USER` field:`e_mail_address` | `CONTACT.EMAIL` x {`COLLECTION`, `STORING`, `USING`} x {`SERVICES`,`COMPLIANCE`} | `CONTRACT` |
+| table:`USER` field:`e_mail_address` | `CONTACT.EMAIL` x `USING` x {`MARKETING`,`ADVERTISING`} | `LEGITIMATE-INTEREST`|
+| table:`USER` field:`e_mail_address` | `CONTACT.EMAIL` x `SHARING` x {`ADVERTISING`} | `CONSENT`|
+
+The records given in the example correspond to the situation of a System that is processing the user's e-mail address for purposes of providing services to the user and for compliance with regulation (KYC) under `CONTRACT` legal base.
```

Comment:
I agree with you. For now, I'm only moving what was in the previous version to a separate doc, because I think it's still useful as it is. I'll review it and at least partially rewrite it in another PR.

Reply:
moved to https://github.com/blindnet-io/blindnet.dev/issues/51